pkgs/development/python-modules/tensorflow-probability/default.nix

```diff
@@ -26,6 +27,8 @@ let
 
   # first build all binaries and generate setup.py using bazel
   bazel-wheel = buildBazelPackage {
+    bazel = bazel_0_26;
```

Comment:
Why are you making these changes? They appear unrelated / unnecessary to the upgrade of Bazel?